src/main/java/seedu/address/model/recommender/Recommendation.java

```diff
@@ -0,0 +1,139 @@
+package seedu.address.model.recommender;
+
+import seedu.address.model.location.Location;
+import seedu.address.model.person.ContactIndex;
+import seedu.address.model.scheduler.time.TimePeriod;
+
+/**
+ * A meetup suggestion consisting of a location and time period.
+ */
+public class Recommendation implements Comparable<Recommendation> {
+    private final Location location;
+    private final TimePeriod timePeriod;
+    private final ContactIndex contactIndex;
+    private final boolean isSaved;
+
```

Comment:
Possible to store a list of people for each recommendation?

Reply:
Possible, just take note of the pitfalls i’ve mentioned in the Telegram.